apps/scan/backend/src/store.ts

```diff
@@ -968,4 +990,46 @@ export class Store {
     if (!result) return undefined;
     return safeParseSystemSettings(result.data).unsafeUnwrap();
   }
+
+  /**
+   * Gets the name of the directory that we're continuously exporting to, e.g.
+   * TEST__machine_SCAN-0001__2023-08-16_17-02-24. Returns undefined if not yet set.
+   */
+  getExportDirectoryName(): string | undefined {
+    const result = this.client.one(
+      'select export_directory_name as exportDirectoryName from export_directory_name'
+    ) as { exportDirectoryName: string } | undefined;
+    return result?.exportDirectoryName;
+  }
+
+  /**
+   * Stores the name of the directory that we'll be continuously exporting to
+   */
+  setExportDirectoryName(exportDirectoryName: string): void {
+    this.client.run('delete from export_directory_name');
+    this.client.run(
+      'insert into export_directory_name (export_directory_name) values (?)',
+      exportDirectoryName
+    );
+  }
+
+  //
+  // Stubs
+  //
```

Comment:
I've stubbed out the methods related to hashes for now since this PR is large enough as is

Issue to track: https://github.com/votingworks/vxsuite/issues/3838